awx/ui_next/src/screens/Schedule/ScheduleList/ScheduleList.jsx

```diff
@@ -0,0 +1,205 @@
+import React, { useState, useEffect } from 'react';
+import { useLocation } from 'react-router-dom';
+import { withI18n } from '@lingui/react';
+import { t } from '@lingui/macro';
+import { SchedulesAPI } from '@api';
+import { Card, PageSection } from '@patternfly/react-core';
+import AlertModal from '@components/AlertModal';
+import ErrorDetail from '@components/ErrorDetail';
+import DataListToolbar from '@components/DataListToolbar';
+import PaginatedDataList, {
+  ToolbarDeleteButton,
+} from '@components/PaginatedDataList';
+import { getQSConfig, parseQueryString } from '@util/qs';
+import { ScheduleListItem } from '.';
+
+const QS_CONFIG = getQSConfig('schedule', {
+  page: 1,
+  page_size: 20,
+  order_by: 'unified_job_template__polymorphic_ctype__model',
+});
+
+function ScheduleList({ i18n }) {
```

Comment:
+1 for making this component name singular (we have a mix & i've been working to make them all consistent)